chocolatey/choco
# (#2860) Add new PowerShell helper function for acquiring paths from private variables

## Description Of Changes

This pull request adds a new PowerShell helper that can be used going forward to get access to the private environment variable `PackagePath`, as well as to get the installation directory of Chocolatey CLI.

## Motivation and Context

There is no direct replacement for using the environment variable `$env:PackagPath` and `$env:ChocolateyPackagePath` that can be used by maintainers (at least on Chocolatey Community Repository.).
As such it makes sense to add this ability to maintainers through the use of a new cmdlet that can be used.

## Testing

1. Mainly tested by running new tests in test kitchen.

But manual testing can be done with:

1. Create a new nupkg package of the source files located in `tests/packages/test-chocolateypath`.
2. Run `choco install <PATH-TO-NUPKG>`
3. Ensure the package outputs the expected values for the package path, and the install path.
4. Run `choco uninstall test-chocolateypath`
5. Ensure the package outputs the expected values for the package path and install path both for the before modify script and the uninstall script.

## Change Types Made


## Related Issue

Fixes #2860

## Change Checklist
